contracts/cw1-whitelist/.cargo/config

```diff
@@ -0,0 +1,6 @@
+[alias]
+wasm = "build --release --target wasm32-unknown-unknown"
+wasm-debug = "build --target wasm32-unknown-unknown"
```

Comment:
Both `wasm` build targets should be performed with `--lib` flag, you will not be able to build schema for wasm. I assume this might be why CI is failing.